api/src/main/scala/uk/ac/wellcome/platform/api/controllers/MainController.scala

```diff
@@ -2,6 +2,7 @@ package uk.ac.wellcome.platform.api.controllers
 
 import com.twitter.finagle.http.Request
 import com.twitter.finatra.http.Controller
+import uk.ac.wellcome.platform.api.models._
```

Comment:
Unintentional. I accidentally removed it and put it back in a different place. I've been following a Python-esque order too, but I'm not too concerned about this class, as it needs removed (along with CalmService) when we start building this out next quarter.